src/styles/blocks/header.scss

```diff
@@ -0,0 +1,61 @@
+@use "../utils/mixins.scss" as underLine;
+@use "../utils/variables" as mainColor;
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding-left: 50px;
+  padding-right: 50px;
```

Comment:
You can use shorthand property here
https://developer.mozilla.org/en-US/docs/Web/CSS/padding#syntax